requirements/requirements.txt

```diff
@@ -37,20 +36,20 @@ django-radius==1.3.3
 django-solo==1.1.3
 django-split-settings==1.0.0
 django-taggit==1.2.0
-django==2.2.10
+django==2.2.10            # channels, django-auth-ldap, django-cors-headers, django-crum, django-jsonfield, django-oauth-toolkit, django-polymorphic, django-taggit, djangorestframework
 djangorestframework-yaml==1.0.3
 djangorestframework==3.11.0
-docutils==0.15.2          # via python-daemon
+docutils==0.16            # via python-daemon
 future==0.16.0            # via django-radius
-gitdb2==2.0.6             # via gitpython
-gitpython==3.0.5
-google-auth==1.10.0       # via kubernetes
+gitdb==4.0.2              # via gitpython
```

Comment:
This is a little weird - did this package name change? cc @AlanCoding

Reply:
yes, the project gitpython changed their dependencies in 3.1.0 https://github.com/gitpython-developers/GitPython/commit/c859019afaffc2aadbb1a1db942bc07302087c52

So the reason you see this package switched is because _this_ PR includes an upgrade of gitpython from 3.0.5 to 3.1.0. I do not have a specific reason to upgrade it at the moment. The tool is not expected to perform such an upgrade unless it is ran with the upgrade option. For the purposes of this PR, I don't have a specific reason to do a general upgrde.